### ISSUE TITLE
Change order of the instructions to users in edit_r_profile and edit_r_environ

### ISSUE THREAD

**Issue by llrs** [r-lib/usethis#732]
When I want to edit my .Rprofile I see this:
```r
edit_r_profile()
● Restart R for changes to take effect
● Modify '/home/lrevilla/.Rprofile'
```
However I think that the order should be the opposite, first Modify then Restart. Similarly the order can be changed in `edit_r_environ`. 

```r
edit_r_profile()
● Modify '/home/llrs/.Rprofile'
● Restart R for changes to take effect
```

I'm happy to submit a small PR if you agree on this

https://github.com/r-lib/usethis/blob/e8fb9d75e1ad0193ce9687a774296f703a9e37a0/R/edit.R#L63

**Comment by jennybc** [comment]
I suspect there's a bit more to think about here, because certain things are arranged they way they are to accommodate context where we can and cannot open the file for editing. But it's certainly OK for you to poke around and see if there's an obvious rationalization.

**Comment by llrs** [comment]
The only reference I found to `edit_r_profile` in `usethis` is in the `use_rprofile_package` function, which affects the `use_reprex`, and other functions of the family rprofile-helper. Which currently suggest to:
```
use_reprex()
● Restart R for changes to take effect
● Modify '/home/llrs/.Rprofile'
● Include this code in '.Rprofile' to make reprex available in all interactive sessions.
  if (interactive()) {
    suppressMessages(require(reprex))
  }
```

Which ideally I think would be easier to read 

```
● Include this code in '.Rprofile' to make reprex available in all interactive sessions.
  if (interactive()) {
    suppressMessages(require(reprex))
  }
● Modify '/home/llrs/.Rprofile'
● Restart R for changes to take effect
```

I couldn't find (but maybe I need to think longer and not on Friday evening) a situation where it cannot be possible to open the file for editing. I'll submit a PR by this weekend.

**Comment by jennybc** [comment]
> I couldn't find (but maybe I need to think longer and not on Friday evening) a situation where it cannot be possible to open the file for editing. I'll submit a PR by this weekend.

We can't assume people are using RStudio or an IDE of any sort.

**Comment by cderv** [comment]
FWIW this change happened in e04499b770010689f8968bf2bb983fe1adb71369 where `edit_r_profile` and `edit_r_environ` were made compatible with environment variable. 
This is where the order has changed. 

I suspect this could be important to restart before modifying for the correct env var to be taken into account. What is misleading here is that, in RStudio, the file open for modification before the user can restart, inviting the user to make the modification. (and opening the file based on the current env var, so my previous point may not be correct 🤔). 

That is also make me wonder : Should usethis todo be seen has ordered or not ? It may be understand has step to be done no matter the order - like a todo list to remember things. ☺️

**Comment by llrs** [comment]
Precisely my issue is that todo bullets are not ordered. It can be confusing to know if I need to restart, edit the Rprofile, restart again, or just one restart before editing or one restart after editing the file. From my understanding of R I think that first I need to edit the file and then restart the R session. That's why I why to reorder the bullets. 

But why should be R restarted to get the right environmental variable?

I used `use_partial_warnings()` outside RStudio in plain R terminal and I can't even see the list of things to do with the new file that gets opened...

**Comment by jennybc** [comment]
I no longer remember why I was so anxious about this. I'll merge and try to play with a bit to make sure I'm not missing something obvious.

**Comment by jennybc** [comment]
Closed by 236c11609ec2f877139a062aad6501cff5c37e93
